src/modules/products/screens/detail-products/details-screen.tsx

```diff
@@ -0,0 +1,105 @@
+import { useNavigation } from '@react-navigation/native';
+import { styled } from 'nativewind';
+import React, { useState } from 'react';
+import { Image as RNImage, TouchableOpacity as RNTouchableOpacity } from 'react-native';
+
+import { common } from '../../../../translations/en.json';
+import DropdownComponent from '../../../../ui/core/select-menu';
+import { Text } from '../../../../ui/core/text';
+import { SafeAreaView, ScrollView, View } from '../../../../ui/core/view';
+const Image = styled(RNImage);
+const TouchableOpacity = styled(RNTouchableOpacity);
+
+const item = {
+  state: 'Restored',
+  name: 'Baumler Chair',
+  manufactur: 'Woodstock Co.',
+  price: 30,
+  description1:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus. Sed dignissim, metus nec fringilla accumsan, risus sem sollicitudin lacus, ut interdum tellus elit sed risus.\n',
+  description2:
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit etiam eu turpis molestie, dictum est a, mattis tellus.',
+};
+
+export const DetailsScreen = () => {
+  const availableItems = 12;
+  const [selectedImage, setSelectedImage] = useState<number>(0);
+  const navigation = useNavigation();
+
+  const images = [
+    require('../../../../ui/assets/images/product-details/image-5.png'),
+    require('../../../../ui/assets/images/product-details/image-6.png'),
+    require('../../../../ui/assets/images/product-details/image-5.png'),
+  ];
+
+  const changePrincipalImage = (index: number) => {
+    setSelectedImage(index);
+  };
+
+  const handleBackToHome = () => {
+    navigation.navigate('Tab');
+  };
+
+  return (
+    <SafeAreaView className="flex-1 bg-gray-100">
+      <ScrollView>
+        <View>
+          <Image
+            style={{ width: '100%' }}
+            source={require('../../../../ui/assets/images/logo-black-market/Header.png')}
+          />
+        </View>
+        <View className="px-4 py-3">
```

Comment:
Why using nativewind on same screens and stylesheet on others?

Reply:
I will use tailwind in all components.